features/search_across_libraries.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Search across all libraries
+  As a researcher, I want to be able to search across all libraries
+  so that I don't need to know the specific focus of a particular library.
```

Comment:
@ekate Yes please feel free to change this however you see fit, but let's also keep in mind that indexing a lot of files every time the test suite is run will continue to become a more and more expensive process. With my next user stories I believe I will start creating cassettes that query the development sold index since we know that we are all looking at the same content. With the data sample size needs of these user stories I think this is the best way.